proto/money/rpc/reservation_folio_svc.proto

```diff
@@ -168,6 +168,7 @@ enum ReservationFolioSvcAuthorizationModificationResult {
 	RESERVATION_FOLIO_SVC_AUTHORIZATION_MODIFICATION_SUCCESS = 0;
 	RESERVATION_FOLIO_SVC_AUTHORIZATION_MODIFICATION_SALE_NOT_FOUND = 1;
 	RESERVATION_FOLIO_SVC_AUTHORIZATION_MODIFICATION_UNKNOWN_FAILURE = 2;
+	RESERVATION_FOLIO_SVC_AUTHORIZATION_MODIFICATION_DECLINED = 3;
```

Comment:
for sure that should get left in

I think almost always though, with these result enums, it should go "success", "unknown failure", then all the enumerated failure cases, since proto does this "enum by number" thing.